plugin/upstream_generator/bq_upstream_generator.go

```diff
@@ -95,18 +102,24 @@ func NewBQUpstreamGenerator(logger log.Logger, parserFunc ParserFunc, bqExtracto
 	if bqExtractorFunc == nil {
 		me.Append(fmt.Errorf("bqExtractorFunc is nil"))
 	}
-	if evaluatorFunc == nil {
-		me.Append(fmt.Errorf("evaluatorFunc is nil"))
+	if len(evaluatorFuncs) == 0 {
+		me.Append(fmt.Errorf("evaluatorFuncs is needed"))
+	}
+	for _, evaluatorFunc := range evaluatorFuncs {
+		if evaluatorFunc == nil {
+			me.Append(fmt.Errorf("evaluatorFunc is nil"))
+			break
```

Comment:
Why we want to break the loop in case a func is nil, we can just ignore this and continue with the rest. ?

Reply:
agree, we could skip if it's nil